src/cockpit/protocol.py

```diff
@@ -269,8 +260,8 @@ async def request_authorization(self, challenge: str, timeout: Optional[int] = N
             self.authorizations.pop(cookie)
 
     def do_authorize(self, message: JsonObject) -> None:
-        cookie = message.get('cookie')
-        response = message.get('response')
+        cookie = get_str(message, 'cookie')
+        response = get_str(message, 'response')
 
         if not isinstance(cookie, str) or not isinstance(response, str):
```

Comment:
This code is unreachable now isn't it? `get_str` would throw an exception?

Reply:
Good catch, thanks!